handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -86,6 +87,15 @@ public Boolean run() {
                     return SystemPropertyUtil.getBoolean("jdk.tls.rejectClientInitiatedRenegotiation", false);
                 }
             });
+
+    private static final int MAX_BIO_SIZE =
+            AccessController.doPrivileged(new PrivilegedAction<Integer>() {
+                @Override
+                public Integer run() {
+                    return Math.min(0, SystemPropertyUtil.getInt("io.netty.handler.ssl.openssl.maxBioSize", 0));
```

Comment:
Yes as it uses the default 17k here then. Let me add some Javadoc explaining it and also make it clear it's about bytes